test/blackbox-tests/test-cases/jsoo/tests.t/run.t

```diff
@@ -1,8 +1,6 @@
 tests stanza with jsoo
 
-  $ dune runtest
-  a: ok
-  b: ok
-  File "run.t", line 1, characters 0-0:
-  Error: Files _build/default/run.t and _build/default/run.t.corrected differ.
+  $ dune build @runtest-js
+  Error: Alias "runtest-js" specified on the command line is empty.
```

Comment:
@rgrinberg, @jeremiedimino how can I convince dune that this alias exists ?

Reply:
You can make it a "standard" alias. See function `make_standard` in `src/dune_engine/alias.ml`.

Reply:
I want to have the alias configurable in env by users,  does it make sense to call make_standard ?

Reply:
Standard only means that if the alias is empty then Dune doesn't complain about it.

Reply:
Thanks